docs/articles/documentation/concepts/browsers.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: docs
+title: Browsers
+permalink: /documentation/concepts/browsers.html
+---
+# Browsers
+
+This topic lists browsers supported by TestCafe and describes how to use various browser features.
+
+* [Browser Support](#browser-support)
+* [Test in Headless Mode](#test-in-headless-mode)
+* [Use Chromium Device Emulation](#use-chromium-device-emulation)
+* [User Profiles](#user-profiles)
+
+## Browser Support
+
+### Officially Supported Browsers
+
+While TestCafe is designed to support most modern browsers, there are a limited number
+of *officially supported* browsers against which TestCafe is actively tested.
+
+* Google Chrome: Stable, Beta, Dev and Canary
+* Internet Explorer (11+)
+* Microsoft Edge (legacy and Chromium-based)
+* Mozilla Firefox
+* Safari
+* Google Chrome mobile
+* Safari mobile
+
+> TestCafe supports the latest version of each browser unless specified explicitly.
+
+### Locally Installed Browsers
+
+TestCafe can automatically detect popular browsers installed on the local computer.
+You can use a short name - *browser alias* - to identify these browsers when you launch tests.
+
+The following table lists browsers that can be detected automatically.
+
+Browser                                    | Browser Alias
+------------------------------------------ | -------------------
+Chromium                                   | `chromium`
+Google Chrome                              | `chrome`
+Google Chrome Canary                       | `chrome-canary`
+Internet Explorer                          | `ie`
+Microsoft Edge (legacy and Chromium-based) | `edge`
+Mozilla Firefox                            | `firefox`
+Opera                                      | `opera`
+Safari                                     | `safari`
+
+Call the `testcafe` command with the [--list-browsers](../reference/command-line-interface.md#-b---list-browsers) flag to view the list of all the available browsers.
+
+To run tests in a different local browser, specify the path to the browser executable file.
+
+> To test in Microsoft Edge Chromium and Legacy on the same machine, [install them side-by-side](https://docs.microsoft.com/en-us/DeployEdge/microsoft-edge-sysupdate-access-old-edge). Once they are installed, see the [Microsoft Edge Legacy Support](#microsoft-edge-legacy-support) section below for information on how to enable testing in older versions.
+
+For more information and examples, see:
+
+* Command line: [Local Browsers](../reference/command-line-interface.md#local-browsers)
+* API: [runner.browsers](../reference/api/runner/browsers.md)
+* Configuration file: [browsers](../reference/configuration-file.md#browsers)
+
+### Portable Browsers
+
+To use a portable browser, specify the path to the browser executable file. For more information and examples, see:
```

Comment:
Change "browser executable file" to "browser's executable file"

For the second sentence, let's say:
For more information and examples, see the following: